src/libraries/System.Runtime.Extensions/tests/System/Numerics/BitOperationsTests.cs

```diff
@@ -269,6 +449,31 @@ public static void BitOps_Log2_ulong(ulong n, int expected)
             Assert.Equal(expected, actual);
         }
 
+        [Theory]
+        [InlineData(0, 0)]
+        [InlineData(1, 0)]
+        [InlineData(2, 1)]
+        [InlineData(3, 2 - 1)]
+        [InlineData(4, 2)]
+        [InlineData(5, 3 - 1)]
+        [InlineData(6, 3 - 1)]
+        [InlineData(7, 3 - 1)]
+        [InlineData(8, 3)]
+        [InlineData(9, 4 - 1)]
+        [InlineData(byte.MaxValue, 8 - 1)]
+        [InlineData(ushort.MaxValue, 16 - 1)]
+        [InlineData(uint.MaxValue, 32 - 1)]
+#if TARGET_64BIT
+        [InlineData(ulong.MaxValue, 64 - 1)]
+        public static void BitOps_Log2_nuint(uint n, int expected)
+#else
+        public static void BitOps_Log2_nuint(ulong n, int expected)
+#endif
+        {
+            int actual = BitOperations.Log2((nuint) n);
+            Assert.Equal(expected, actual);
```

Comment:
Why is this 64?

> ```
>     System.Numerics.Tests.BitOperationsTests.BitOps_TrailingZeroCount_nint(n: 0, expected: 32) [FAIL]
>       Assert.Equal() Failure
>       Expected: 32
>       Actual:   64
>       Stack Trace:
>         /_/src/libraries/System.Runtime.Extensions/tests/System/Numerics/BitOperationsTests.cs(408,0): at System.Numerics.Tests.BitOperationsTests.BitOps_TrailingZeroCount_nint(Int32 n, Int32 expected)
> ```
> Test Runner Output:     System.Numerics.Tests.BitOperationsTests.BitOps_TrailingZeroCount_nuint(n: 0, expected: 32) [FAIL]
      Assert.Equal() Failure
      Expected: 32
      Actual:   64
      Stack Trace:
        /_/src/libraries/System.Runtime.Extensions/tests/System/Numerics/BitOperationsTests.cs(366,0): at System.Numerics.Tests.BitOperationsTests.BitOps_TrailingZeroCount_nuint(UInt32 n, Int32 expected)